server/src/main/java/net/cryptic_game/backend/server/server/http/endpoints/HttpInfoEndpoint.java

```diff
@@ -18,7 +19,7 @@ public HttpInfoEndpoint(final DaemonHandler daemonHandler) {
 
     @ApiEndpoint("status")
     public ApiResponse status() {
-        return new ApiResponse(ApiResponseType.OK, JsonUtils.toArray(this.daemonHandler.getDaemons()));
+        return new ApiResponse(ApiResponseType.OK, JsonUtils.toArray(this.daemonHandler.getDaemons(), Daemon::serialize));
```

Comment:
You dont need `JsonUtils#toArray(Collection<?>)`. The new `ApiResponse` can hadle `Collections` without any help.